src/engine/source/kvdb/include/kvdb/kvdb.hpp

```diff
@@ -0,0 +1,8 @@
+#ifndef _KVDB_H
+#define _KVDB_H
+#include <functional>
```

Comment:
We should clean this include until they are really needed.

Reply:
done